recipes/stempeg/LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
Can you remove/revert the delete commits of these files. They're handled automatically by the CI and deleting them can cause problems.

Reply:
what should i do?
recovery whole stempeg folder?

Reply:
This is what I would suggest but take care that these are destructive commands that can delete changes you have locally.
```bash
# Delete the last 4 commits
git reset --hard HEAD~4
# Add the two commits you want
git cherry-pick 0edd7cae9f2eb0dca8fded37fd7905e5ce413688 9b165c8e6d492da739de0cb95d83cc086206196c
# Force push as the history has now diverged
git push -u origin master -f
```

Reply:
I think its done